source/index.txt

```diff
@@ -134,3 +134,24 @@ To's and Articles page <https://www.mongodb.com/developer/learn/?content=Article
 
 To ask questions and engage in discussions with fellow developers using
 the Kotlin Driver, see the `forums page <https://www.mongodb.com/community/forums/tag/kotlin>`__.
+
+.. TODO: Add back when University content is available for Kotlin driver
+.. need to re-add /includes/figures/M220J_hero.jpg
+
+.. MongoDB University
```

Comment:
Since this is commented out, change lines 129-130 to omit MDB university reference